lib/ios/TabBarItemAppearanceCreator.m

```diff
@@ -13,13 +13,25 @@ + (void)setTitleAttributes:(UITabBarItem *)tabItem titleAttributes:(NSDictionary
     tabItem.standardAppearance.stackedLayoutAppearance.normal.titleTextAttributes = titleAttributes;
     tabItem.standardAppearance.compactInlineLayoutAppearance.normal.titleTextAttributes = titleAttributes;
     tabItem.standardAppearance.inlineLayoutAppearance.normal.titleTextAttributes = titleAttributes;
+    if (@available(iOS 14.0, *)) {
+      UIColor *color = [titleAttributes valueForKey:@"NSColor"];
```

Comment:
Please replace `@"NSColor"` with the existing constant `NSForegroundColorAttributeName`